src/Symfony/Component/DependencyInjection/Attribute/Autowire.php

```diff
@@ -52,6 +58,8 @@ public function __construct(
         $this->value = match (true) {
             null !== $service => new Reference($service),
             null !== $expression => class_exists(Expression::class) ? new Expression($expression) : throw new LogicException('Unable to use expressions as the Symfony ExpressionLanguage component is not installed. Try running "composer require symfony/expression-language".'),
+            null !== $env => new EnvConfigurator($env),
+            null !== $param => new ParamConfigurator($param),
```

Comment:
```
            null !== $env => "%env($env)%",
            null !== $param => "%$param%",
```

Reply:
I took inspiration from this function `\Symfony\Component\DependencyInjection\Loader\Configurator\param`.

I feel that it is better not to smear the logic of parameter and env variable formatting over the code. Of course, this is unlikely to change in the future, but still.

If you don't mind, I want to leave classes.

Reply:
I do mind sorry. Those classes are meant to be used when using the PHP-DSL. They're unrelated to this attribute.

Reply:
Refactored.